src/site/_includes/partials/post-next.njk

```diff
@@ -0,0 +1,184 @@
+            {% if updated %} — {{ 'i18n.common.updated' | i18n((locale)) }} <time>{{ updated | prettyDate }}</time> {% endif %}
+          </div>
+        {% endif %}
+
+        {% set languageListHtml %}{% LanguageList page.url, lang %}{% endset %}
+        {% if languageListHtml %}
+        <div class="flow">
+          {{ languageListHtml | safe }}
+        </div>
+        {% endif %}
+
+        {% SignPosts page.fileSlug %}
+
+        {% if authors %}
+          <div class="cluster flow-space-size-2">
+            {% for author in authors %}
+              {% Author {
+                id: authorKey,
+                author: collections.authors[author],
+                showSocialMedia: true,
+                locale: locale
+                }
+              %}
+            {% endfor %}
+          </div>
+        {% endif %}
+      </header>
+
+      {% if draft and site.env === 'dev' %}
+        <div class="banner bg-state-warn-bg color-core-text">
+          <p><strong>{{ 'i18n.post.is_draft' | i18n(locale) }}</strong></p>
+        </div>
+      {% endif %}
+
+      {% include 'partials/toc-inner.njk' %}
+
+      {{ content | safe }}
+
+      {% if stack_overflow_tag %}
+        {% StackOverflow stack_overflow_tag %}
+      {% endif %}
+
+      <div class="cluster gutter-base flow-space-size-2" role="list" aria-label="tags">
+        {% for tagKey in tags or [] %}
+          {% if tagKey in collections.tags %}
+            {% set tag = collections.tags[tagKey] %}
+            <a class="pill" href="{{tag.url}}">{{tag.overrideTitle or tag.title | i18n(locale)}}</a>
+          {% endif %}
+        {% endfor %}
+      </div>
+
+      <div class="text-size-0 color-mid-text">
+        <span>
+          {% if updated %}
+          {{ 'i18n.post.last_updated' | i18n(locale) }}: <time>{{ updated | prettyDate }}</time>
+          {% else %}
+          {{ 'i18n.post.last_updated' | i18n(locale) }}: <time>{{ date | prettyDate }}</time>
+          {% endif %}
+        </span>
+        —
+        <a
+          href="{{ page.inputPath | githubLink }}"
+        >
+          {{ 'i18n.post.improve_article' | i18n(locale) }}
+        </a>
+      </div>
+
+      {% if codelabs %}
+        <div class="flow-space-size-2">
+          {% CodelabsCallout codelabs, lang %}
+        </div>
+      {% endif %}
+
+      {% if 'blog' in tags %}
+        {% set backUrl = '/blog' %}
+        {% set backLabel = 'Return to all articles' %}
+      {% elif path %}
+        {% set backUrl = '/' + path.slug %}
+        {% set backLabel = 'Return to all articles' %}
+      {% endif %}
+
+      {% if backLabel and backUrl %}
+        <div class="flow-space-size-2">
+          <a href="{{ backUrl }}" class="button" data-type="secondary">
+            {% include 'icons/arrow-back.svg' %}
+            <span>{{ backLabel }}</span>
+          </a>
+        </div>
+      {% endif %}
+
+      {% include 'partials/subscribe-action-next.njk' %}
```

Comment:
Share action is missing.
Subscribe action needs to be commented out (we have disabled the newsletter atm)